style.css

```diff
@@ -60,6 +60,173 @@ header {
   background-color: lightcyan;
 }
 
-ul li {
-  display: inline;
+ul {
+  list-style: none;
+}
+
+ul.social {
+  display: flex;
+  padding: 0;
+  justify-content: flex-start;
+}
+
+div.worksec {
+  display: grid;
+  grid-template-columns: 1fr 375px 1fr;
+}
+
+header.work {
+  padding: 60px 0 0 0;
+}
+
+article.work {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  margin: 40px auto;
+  grid-column-start: 2;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+article.AboutMe {
+  grid-column-start: 2;
+}
+
+h1.work {
+  width: 327px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+img.work {
+  width: 395px;
+  height: 290px;
+}
+
+ul.worklist {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+}
+
+li.card-title {
+  width: 55px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  padding: 0 10px;
+}
+
+li.work {
+  width: 98px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  padding: 0 10px;
+}
+
+li.counter {
+  padding: 0 10px;
+}
+
+p.work {
```

Comment:
- [OPTIONAL ] please I would recommend you to just use only classes as CSS selectors without the prefix tag to make your work more expressive and standard. instead of writing `p.work`, you can write `.work`